Grand Circus - OOP/GrandCircus/CircusModel/AnimalBase.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace iQuest.GrandCircus.CircusModel
+{
+    internal abstract class AnimalBase : IAnimal
+    {
+        private readonly string name;
+        private readonly string speciesName;
+
+        public AnimalBase(string name,string speciesName)
+        {
+            this.name = name;
+            this.speciesName = speciesName;
+        }
+        
+
+        public string GetName()
```

Comment:
Instead of using methods just to return a value, we should use properties: https://www.w3schools.com/cs/cs_properties.php.
To not be accessible from external classes, the properties should be get-only. public string Name { get; }
To make sure the class contains the properties, please declare them also in the IAnimal interface.